app-backend/tool/src/main/scala/eval/Interpreter.scala

```diff
@@ -92,10 +94,9 @@ object Interpreter extends LazyLogging {
     * without fetching any Rasters. Only interprets the structural validatity of
     * the AST, given the params.
     */
-  def interpretPure[M: Monoid](ast: MapAlgebraAST, params: EvalParams): Interpreted[M] = ast match {
-    case Source(id, label) if params.sources.isDefinedAt(id) => Valid(Monoid.empty)
-    case Source(id, _) => Invalid(NonEmptyList.of(MissingParameter(id)))
```

Comment:
You're 100% right, I must have read the `if` as being on the right side or something